app/controllers/katello/api/v2/content_view_versions_controller.rb

```diff
@@ -83,42 +82,37 @@ def republish_repositories
-    param :destination_server, String, :desc => N_("Destination Server name"), :required => false
-    param :organization_id, :number, :desc => N_("Organization identifier"), :required => false
-    param_group :search, Api::V2::ApiController
-    add_scoped_search_description_for(ContentViewVersionExportHistory)
-    def export_histories
-      history = ContentViewVersionExportHistory.readable
-      history = history.where(:content_view_version_id => params[:content_view_version_id]) unless params[:content_view_version_id].blank?
-      history = history.where(:destination_server => params[:destination_server]) unless params[:destination_server].blank?
-      history = history.with_organization_id(params[:organization_id]) unless params[:organization_id].blank?
-      history = history.with_content_view_id(params[:content_view_id]) unless params[:content_view_id].blank?
-      respond_for_index(:collection => scoped_search(history, 'id', 'asc', resource_class: ContentViewVersionExportHistory),
-                        :template => '../../../api/v2/content_view_version_export_histories/index')
-    end
-
-    api :GET, "/content_view_versions/export_api_status", N_("true if the export api is pulp3 ready and usable. This API is intended for use by hammer-cli only.")
-    def export_api_status
-      ::Foreman::Deprecation.api_deprecation_warning("export_api_status is being deprecated and will be removed in a future version of Katello.")
-      render json: { api_usable: SmartProxy.pulp_primary.pulp3_repository_type_support?(Katello::Repository::YUM_TYPE) }, status: :ok
-    end
-
-    api :POST, "/content_view_versions/:id/export", N_("Export a content view version")
+    api :POST, "/content_view_versions/:id/export_legacy", N_("Export a content view version.  Relevant only for Pulp 2 repositories.")
     param :id, :number, :desc => N_("Content view version identifier"), :required => true
-    param :destination_server, String, :desc => N_("Destination Server name, required for Pulp3")
-    param :chunk_size_mb, :number, :desc => N_("Chunk export-tarfile into pieces of chunk_size mega bytes. Relevant only for Pulp 3 repositories"), :required => false
-    param :export_to_iso, :bool, :desc => N_("Export to ISO format. Relevant only for Pulp 2 repositories"), :required => false
+    param :export_to_iso, :bool, :desc => N_("Export to ISO format."), :required => false
     param :iso_mb_size, :number, :desc => N_("maximum size of each ISO in MB. Relevant only for Pulp 2 repositories"), :required => false
     param :since, Date, :desc => N_("Optional date of last export (ex: 2010-01-01T12:00:00Z). Relevant only for Pulp 2 repositories"), :required => false
-    def export
-      task = if SmartProxy.pulp_primary.pulp3_repository_type_support?(Katello::Repository::YUM_TYPE)
-               export_pulp_v3
-             else
-               export_pulp_v2
-             end
+    def export_legacy
```

Comment:
If this controller method name was `export` before we added the new stuff, I think it should stay that way rather than renaming it. Let me know if I'm missing something!

Reply:
@parthaa do you agree or disagree here?